cgroups/fs/apply_raw.go

```diff
@@ -58,7 +58,7 @@ type subsystem interface {
 	// Removes the cgroup represented by 'data'.
 	Remove(*data) error
 	// Creates and joins the cgroup represented by data.
-	Set(*data) error
+	Apply(*data) error
```

Comment:
do you think `Join` is a better name since the comments on various places refers to the action as joining the cgroups ?